account.js

```diff
@@ -36,7 +38,7 @@ class BankAccount {
   }
 
   withdraw(withdrawAmount, accountNumber) {
-    if (this.accountNumber === accountNumber && this.accountStatus === 'active') {
+    if (this.accountNumber === accountNumber && this.IsActive()) {
```

Comment:
it is always not a good idea to start a method name with a capital letter, rename `IsActive` to `isActive`